### PR TITLE
Fixed "photos" and "health_kit" tests for non-english languages.

### PR DESCRIPTION
**Pull request by vsafonkin**
Problem: "photos" and "health_kit" tests have queries with english words, it's cause fails these tests with non-english languages.

Solution: make queries not depending from languages.

### CODE REVIEW

File: features/00-permissions.feature [hunk 1]
```diff
@@ -22,7 +22,6 @@ When I touch the Background Location Services row
 Then Calabash should dismiss the alert
 And background location services are authorized
 
-@skip_for_non_english
```
Comment (ivan-nosar) [inline features/00-permissions.feature]: I believe we also need to remove `Before("@skip_for_non_english")` in `features/support/01_launch.rb` file

Comment (vsafonkin) [reply features/00-permissions.feature]: Done!

**Comment by jmoody** [comment]
There are variety of failures in CI.  I think the Xcode 11.1/Mojave failures are because the simulators are in a really bad state.

The Xcode 11.2 failures are more interesting:

```
Failing Scenarios:
cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:23 # Scenario Outline: Alerts in non-portrait orientation, Examples (#1)
cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:24 # Scenario Outline: Alerts in non-portrait orientation, Examples (#2)
cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:25 # Scenario Outline: Alerts in non-portrait orientation, Examples (#3)
```

Please confirm that these tests work on Simulators locally and then merge.

**Comment by vsafonkin** [comment]
> There are variety of failures in CI. I think the Xcode 11.1/Mojave failures are because the simulators are in a really bad state.
> 
> The Xcode 11.2 failures are more interesting:
> 
> ```
> Failing Scenarios:
> cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:23 # Scenario Outline: Alerts in non-portrait orientation, Examples (#1)
> cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:24 # Scenario Outline: Alerts in non-portrait orientation, Examples (#2)
> cucumber -p simulator_vars -p simulator_tags -p formatter features/01-orientation.feature:25 # Scenario Outline: Alerts in non-portrait orientation, Examples (#3)
> ```
> 
> Please confirm that these tests work on Simulators locally and then merge.

Small clarification: non-portrait tests fails in the Xcode 11.1 and these tests does not contain "photos" and "health" scenarios.  Now we're investigating non-portrait failures.

**Pull request merged**